manifests/e/EpicGames,Inc/EpicGamesLauncher/1.2.17.0/EpicGames,Inc.EpicGamesLauncher.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: EpicGames,Inc.EpicGamesLauncher

```

Comment:
This is a duplicate of `EpicGames.EpicGamesLauncher`.

Reply:
bruh i searched, and it wasn't there, what should i do, tell microsoft, or anyone, who knows how to fix that.

Reply:
Could you post a screenshot of your search? What you can do is add tags so its easier to find.

Reply:
ok found it, i searched games store, and it is supposed to be games launcher, so yeah i am sorry for the rude behaviour.

Reply:
Tell me the tags you would like to find the Epic Games Store by and ill add them once i get around to it, the moniker is egs so you might be right that it should be Epic Games Store, however Epic Games Launcher seems like the name it has in Windows unless its incorrect.

Reply:
Fixed in #16208, you can add more tags if need be.

Reply:
how do you add tags?